components/up-job.libsonnet

```diff
@@ -38,6 +38,9 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           '--latency=10s',
           '--initial-query-delay=5s',
           '--threshold=0.90',
+          '--tls-ca-file=' + up.config.withTLS.clientCAFile,
```

Comment:
This should really be called `serverCAFile` because it's the CA that signed the _server's_ certificate that matters here, not the client's